install/wipe_ncn_disks_for_reinstallation.md

```diff
@@ -228,34 +262,83 @@ RAIDs, zeroing the disks, and then wiping the disks and RAIDs.
+      > **Note:** The following output  means the etcd volume  mapper is not present.
+      ```bash
+      No device specified.
+      Command failed.
+      ```
 
-    ```bash
-    ncn# sgdisk --zap-all /dev/sd*
-    ncn# wipefs --all --force /dev/sd* /dev/disk/by-label/*
-    ```
+1. Remove etcd Volumes **on master nodes ONLY**.
+
+   ```bash
+   ncn-m# vgremove etcdvg0
+   ```
+
+1. Remove metal LVM.
+
+     ```bash
+     ncn# vgremove -f -v --select 'vg_name=~metal*'
+     ```
+
+   > **NOTE:** Optionally, run the `pvs` command. If any drives are still listed, remove them with `pvremove`, but this is rarely needed. Also, if the above command fails or returns a warning about the filesystem being in use, ignore the error and proceed to the next step, as this will not inhibit the wipe process.
+
+1. Group these commands together for each node.
+
+   This group of commands should be done in succession on one node before moving to do the same set of commands on the next node. The nodes would be addressed in descending order for each type of node.  Start with the utility storage nodes, then the worker nodes, then ncn-m003, then ncn-m002. 
+
+   > **WARNING:** Do not run these commands on ncn-m001
+
+   1. Stop the RAIDs.
+
+       ```bash
+       ncn# for md in /dev/md/*; do mdadm -S -v $md || echo nope ; done
+       ```
+
+   1. List the disks for verification.
+
+       ```bash
+       ncn# ls -1 /dev/sd* /dev/disk/by-label/*
+       ```
+
+   1. Wipe the disks and RAIDs.
+
+       ```bash
+       ncn# wipefs --all --force /dev/sd* /dev/disk/by-label/*
+       ```
+
+       If any disks had labels present, output from `wipefs` looks similar to the following:
 
-   **NOTE:** On worker nodes, it is a known issue that the `sgdisk` command sometimes encounters a hard hang. If there is no output from the command for 90 seconds, close the terminal session to the worker node, open a new terminal session to it, and complete the disk wipe procedure by running the above `wipefs` command.
+       ```
+       /dev/sda: 8 bytes were erased at offset 0x00000200 (gpt): 45 46 49 20 50 41 52 54
+       /dev/sda: 8 bytes were erased at offset 0x6fc86d5e00 (gpt): 45 46 49 20 50 41 52 54
+       /dev/sda: 2 bytes were erased at offset 0x000001fe (PMBR): 55 aa
+       /dev/sdb: 6 bytes were erased at offset 0x00000000 (crypto_LUKS): 4c 55 4b 53 ba be
+       /dev/sdb: 6 bytes were erased at offset 0x00004000 (crypto_LUKS): 53 4b 55 4c ba be
+       /dev/sdc: 8 bytes were erased at offset 0x00000200 (gpt): 45 46 49 20 50 41 52 54
+       /dev/sdc: 8 bytes were erased at offset 0x6fc86d5e00 (gpt): 45 46 49 20 50 41 52 54
+       /dev/sdc: 2 bytes were erased at offset 0x000001fe (PMBR): 55 aa
+       ```
 
-   See [Basic Wipe](#basic-wipe) section for expected output from the `wipefs` command.
+       Verify there are no error messages in the output.
 
+       The `wipefs` command may fail if no labeled disks are found, which is an indication of a larger problem.
```

Comment:
The comment I left on +326 would mitigate false-positives on disks, but not missing labels.

Reply:
Not sure what change you are requesting.